validator/pandora/client.go

```diff
@@ -0,0 +1,139 @@
+package pandora
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/pkg/errors"
+)
+
+// PandoraClient is a wrapper of rpc client
+type PandoraClient struct {
+	c *rpc.Client
+}
+
+// GetWorkResponseParams contains response params of `eth_gethWork` api
+type GetWorkResponseParams struct {
```

Comment:
maybe adjusting `ctrl + alt + l` or `cmd + alt + l` in Mac

Reply:
Thanks i have forgot to run `go fmt` command.